css/main.css

```diff
@@ -0,0 +1,964 @@
+}
+
+input, textarea {
+  height: 3em;
+  width: 80%;
+  font-size: .80rem;
+  color: #fff;
+  border: 1px solid #7e3218;
+  margin-bottom: 5%;
+  background-color: #7e3218;
+  padding: 0 .7em;
+  display: inline;
+}
+
+.s-form-message-field {
+  height: 15em;
+  padding-top: 1em;
+  color: #fff;
+}
+
+::-webkit-input-placeholder {
+  color: #fff;
+}
+
+:-moz-placeholder {
+  color: #fff;
+}
+
+::-moz-placeholder {
+  color: #fff;
+}
+
+:-ms-input-placeholder {
+  color: #fff;
+}
+
+.s-form-btn {
+  background-color: #fff;
+  color: #7e3218;
+  height: 3em;
+}
+
+.s-theme-author {
+  text-decoration: none;
+  color: #fd6632;
+}
+
+.s-theme-author:hover {
+  color: #0b9ce0;
+}
+
+/*=== BREAKPOINTS ===*/
+@media only screen and (min-width: 667px) {
+
+  h1 {
+    font-size: 2.4em;
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  article {
+    padding: 2em 10%;
+  }
+
+  .l-flex-container {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -o-flex;
+    display: -webkit-flex;
+    display: flex;
+    -moz-flex-direction: row;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    -moz-flex-flow: row wrap;
+    -ms-flex-flow: row wrap;
+    flex-wrap: wrap;
+    -webkit-justify-content: center;
+    justify-content: center;
+    -webkit-align-content: center;
+    align-content: center;
+    -webkit-align-items: center;
+    align-items: center;
+  }
+
+  #portfolio {
+    padding:2em 9%;
+  }
+
+  .l-portfolio-flex {
+    -webkit-flex-flow: row wrap;
+  }
+
+  .h3-lines:before, .h3-lines:after {
+    width: 20%;
+  }
+
+  h2:before, h2:after {
+    width: 30%;
+  }
+
+  .icon-social {
+    text-decoration: none;
+    font-size: 1rem;
+    padding: 0%;
+  }
+
+  .nav-toggle {
+    display: none;
+  }
+
+  .trigger {
+    display: none;
+  }
+
+  .l-nav {
+    left: 0em;
+    width: 100%;
+    height: 3.5em;
+    background: #ffffff;
+    opacity: 1;
+    z-index: 5;
+  }
+
+  .main-nav {
+    display: block;
+  }
+
+  .nav-link {
+    display: inline-block;
+    text-align: center;
+    padding-top: 0;
+    padding: 0 1%;
+  }
+
+  .nav-link a {
+    text-decoration: none;
+    color: #2e2e2e;
+    font-size: .85em;
+    padding: 1%;
+  }
+
+  .nav-link a:active {
+    color: #fd6632;
+  }
+
+   .nav-link a:hover {
+    color: #fd6632;
+  }
+
+  .slider-break {
+    display: block;
+    width: 100%;
+  }
+
+  .slider {
+    display: none;
+  }
+
+  .hero-image {
+    background-image: url("/img/Background.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .s-hero-text {
+    padding: 0 .5em;
+    font-size: 3.25em;
+    line-height: 1.5;
+  }
+
+  .l-subnav {
+    display: block;
+  }
+
+  .l-subnav-link {
+    display: inline-block;
+  }
+
+  .s-portfolio-filter-links {
+    width: auto;
+    padding: 1%;
+  }
+
+  .s-contact-info {
+    display: inline-block;
+    background-image: url("/img/map_base.png");
+    background-repeat: no-repeat;
```

Comment:
Prefer the short-hand here: `background: url("/img/map_base.png") no-repeat;`.

Also, your image filenames are inconsistent - some are lowercase, some start with an uppercase letter.